src/index.js

```diff
@@ -49,20 +42,42 @@ const environmentMap = {
 
 async function init() {
   try {
-    const {
-      environment,
-      bridgeType,
-      templateName,
-      jsPath
-    } = await inquirer.prompt(promptConfig);
+    const { environment, bridgeType, templateName } = await inquirer.prompt(
+      promptConfig
+    );
+
+    const fileTypes = ["JS"];
+    const includediOS = environment.find(
+      answer => answer.indexOf("iOS") !== -1
+    );
+    if (includediOS) fileTypes.unshift("iOS");
+    const includedAndroid = environment.find(
+      answer => answer.indexOf("Android") !== -1
+    );
+    if (includedAndroid) fileTypes.unshift("Android");
+
+    const extraPromptConfig = fileTypes.map(fileType => {
+      return {
+        type: "input",
+        name: `${fileType.toLowerCase()}Path`,
+        message: `What directory should we deliver your ${fileType} files to?`,
+        default: fileType === "JS" ? "." : fileType.toLowerCase(),
```

Comment:
For the default, we should create an object above with the file types as the keys and the default paths as the values. Then, you can look them up.

You can use the paths that we're currently delivering the files to in RNCB as your defaults.